README.md

```diff
@@ -62,8 +62,10 @@ The video will sync automatically when the host plays/pauses the video or seeks
 
 ### To start the server locally : 
 
- 1. Open the "server" folder that we downloaded earlier.
- 2. Open command line there and run this command :    **node index.js**  
+ 1. Install node on your local from this link :  https://nodejs.org/en/download/
+ 2. Open command line and install npm using this command : **npm install -g npm**
```

Comment:
I meant just add the command "npm install" so that we install all the dependencies

Reply:
I have made the changes..Please review @chaitanyabisht